sources/10MBDRIVER

```diff
@@ -74,47 +73,41 @@ Copyright (c) 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1990, 1991 by Venue & Xe
              PACKET])
 
 (\10MBSENDPACKET
-  [LAMBDA (NDB PACKET)                                   (* ; "Edited 14-Apr-88 15:30 by bvm")
```

Comment:
ETHERLIGHTNING is really handy for debugging behavior in flaky network situations.   Please don't take this out.